build.gradle

```diff
@@ -90,22 +84,39 @@ dependencies {
     implementation "org.grails:grails-plugin-url-mappings"
     implementation "org.grails:grails-plugin-interceptors"
     implementation "org.grails.plugins:cache"
+    implementation "org.grails.plugins:cache-ehcache:3.0.0"
     implementation "org.grails.plugins:async"
     implementation "org.grails.plugins:scaffolding"
     implementation "org.grails.plugins:events"
-    implementation "org.grails.plugins:hibernate5"
-    implementation "org.hibernate:hibernate-core:5.4.18.Final"
+    implementation "org.grails.plugins:hibernate5:${gormVersion}"
+    implementation "org.hibernate:hibernate-core:5.6.15.Final"
+    implementation "org.hibernate:hibernate-ehcache:5.4.9.Final"
+    implementation "org.grails.plugins:scaffolding"
     implementation "org.grails.plugins:gsp"
+    implementation "org.codehaus.groovy:groovy-all:your-groovy-version"
+    implementation "io.micronaut:micronaut-core:3.10.2"
+    implementation "io.micronaut:micronaut-inject:3.10.2"
```

Comment:
I do not think these 3 are required. groovy-all, micronaut-core, micronaut-inject

Reply:
Fixed, committed .